lib/data/entity.js

```diff
@@ -282,7 +282,7 @@ const selectFields = (entity, properties, selectedProperties) => {
     createdAt: entity.createdAt,
     creatorId: entity.aux.creator.id.toString(),
     creatorName: entity.aux.creator.displayName,
-    updates: entity.aux.stats.updates,
+    updates: entity.updates, // 'updates' is a property predating 'version' and will always equal version - 1
```

Comment:
I feel like the comment here isn't super necessary, since there's already a comment in the `Entity` frame.